paywall/src/data-iframe/blockchainHandler/locksmithTransactions.js

```diff
@@ -20,16 +20,18 @@ export async function storeTransaction({
   await ensureWalletReady(walletService)
   const account = getAccount()
   const network = getNetwork()
+  // we use the transaction lock as the recipient
+  const recipient = transaction.lock
 
   const url = `${locksmithHost}/transaction`
 
   const payload = {
     transactionHash: transaction.hash,
-    sender: account.toLowerCase(),
+    sender: account,
```

Comment:
this is the one place it is unnecessary. locksmith converts all addresses to their checksummed addresses anyways (which is the other, better way to normalize ethereum addresses, but it requires ethers)